packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -263,15 +245,13 @@ class FlutterDriver {
         isolate.pauseEvent is! VMPauseExceptionEvent &&
         isolate.pauseEvent is! VMPauseInterruptedEvent &&
         isolate.pauseEvent is! VMResumeEvent) {
-      await Future<void>.delayed(_shortTimeout(timeoutMultiplier) ~/ 10);
```

Comment:
Another unexplained delay.  This was a smell to begin with, but I'm curious why it existed and if we're going to re-introduce an old problem by removing it without correspondingly fixing some underlying issue.

Reply:
Quite possible, but I've no idea what to do about it. Leaving the delay in doesn't seem reasonable either, it's just making the original bug harder to track down.